File: discovery-service/src/main/java/com/sudo248/discoveryservice/controller/dto/SupplierDto.java
package com.sudo248.discoveryservice.controller.dto;

import com.sudo248.discoveryservice.repository.entity.Location;
import lombok.AllArgsConstructor;
import lombok.Data;
import lombok.NoArgsConstructor;

import java.util.List;

@Data
@NoArgsConstructor
@AllArgsConstructor
public class SupplierDto {
    private String supplierId;
    private String name, avatar;
    private Location location;
    private List<SupplierProductDto> supplierProducts;
}

File: client/SocStaff/app/src/main/java/com/sudo248/soc_staff/data/dto/discovery/SupplierDto.kt
package com.sudo248.soc_staff.data.dto.discovery

import com.sudo248.soc_staff.domain.entity.user.Location

data class SupplierDto(
    val supplierId: String = "",
    val name: String,
    val avatar: String,
    val location: Location? = null,
    val supplierProducts: List<SupplierProductDto>? = null
)
